packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -354,7 +354,9 @@ export function buildRootObject(powers, _params, testJigSetter = undefined) {
           details`invitations must have a description string: ${description}`,
         );
 
-        const invitationHandle = /** @type {InvitationHandle} */ (harden({}));
+        const invitationHandle = /** @type {InvitationHandle} */ (makeHandle(
+          'InvitationHandle',
+        ));
```

Comment:
```suggestion
        const invitationHandle = makeHandle('Invitation');
```

And possibly need to `@typedef {Handle<'Invitatiton'>} InvitationHandle` if it doesn't work without the typecast.